sweep.py

```diff
@@ -59,7 +59,7 @@
 def Bb(ix):
     return Matrix([[0,dNNx[ix],0],[0,0,dNNy[ix]],[0,dNNy[ix],dNNx[ix]]])
 def Bs(ix):
-    return Matrix([[dNNx[ix],-NN[ix],0],[dNNy[ix],0,-NN[ix]]])
+    return Matrix([[dNNx[ix],-dNNy[ix],0],[dNNy[ix],0,-dNNx[ix]]])
```

Comment:
Components of B matrix must be derivatives all?
Replace NN with dNNx or dNNy based on some textbooks.